ts.config.js

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "module": "commonjs",
```

Comment:
I'm guessing we want `"module": "es6"` unless there is a reason not to. I know Webpack gets fussy about code splitting if you don't use es6 modules, but I'm not sure about rollup.

Reply:
We're using commonjs in the rollup config so I figured it would be the same here!

Reply:
https://github.com/primer/components/blob/master/rollup.config.js#L5

Reply:
I don't quite remember why that was needed though 🤔

Reply:
Either should work. Is there a reason you left it as a js file? Should be `tsconfig.json` although I assume TypeScript accepts both. `.json` is standard though.

Reply:
Fixed!